onnxruntime/core/dlpack/dlpack_converter.h

```diff
@@ -10,13 +10,28 @@
 
 namespace onnxruntime {
 namespace dlpack {
-    
+
+typedef struct {
+  OrtValue handle;
+  DLManagedTensor tensor;
+} OrtDLManagedTensor;
+
+typedef void DlpackDeleterFct(DLManagedTensor* arg);
+
+DlpackDeleterFct* GetDlpackDeleter();
+
 // This convertor will take an OrtValue and wrap it as a DLPack tensor
 // This may create a new ownership to the underlying tensor in OrtValue,
 // so we do pass-by-value here. We don't use pass-by-reference because
 // it implies no new ownership.
 DLManagedTensor* OrtValueToDlpack(OrtValue& ort_value);
 
+// This convertor will take an OrtValue and wrap it as a DLPack tensor,
+// similar to the previous function but the structure DLManagedTensor
+// is already allocated. This function can be used when multiple
+// OrtValue are converted in the same function. This saves allocations.
+DLManagedTensor* OrtValueToDlpack(OrtValue& ort_value, OrtDLManagedTensor* ort_dlmanaged_tensor, DlpackDeleterFct* deleter);
```

Comment:
This one is not used in your latest change? And all the changes in dlpack_converter.h/cc are all related to this new added function?

Reply:
Removed.